openlibrary/templates/books/custom_carousel.html

```diff
@@ -8,10 +8,12 @@ <h2 class="home-h2"><a href="$url">$title</a></h2>
         </div>
       <div class="carousel-container carousel-container-decorated">
         <div class="carousel carousel-$key">
+          $# fallback for users w/ js disabled
+          $ max_visible_books = 5
```

Comment:
should this be in ctx so it can be shared across templates ?

Reply:
that could make sense, within ~ plugins/openlibrary/code.py

Reply:
LGTM with suggested change.